### ISSUE TITLE
Can't update discord canary to 0.0.36

### ISSUE THREAD

**Issue by FreeSlave** [crmarsh/discord-linux-bugs#247]
Upon launching discord canary it says that new version is available. Clicking the Download button opens https://discordapp.com/api/download/undefined?platform=linux&format=deb link in browser and it renders as json:
```json
{
code: 0,
message: "404: Not Found"
}
```

**Comment by OvermindDL1** [comment]
Same issue, I had to download the installer itself from the main website instead.

**Comment by sfirfi** [comment]
The bug got reported to Trello: https://trello.com/c/ETtd68fO
Lets hope it gets fixed soon. Here are the download links for both canary versions:
https://discordapp.com/api/download/canary?platform=linux&format=deb
https://discordapp.com/api/download/canary?platform=linux&format=tar.gz

**Comment by sfirfi** [comment]
This should be fixed with the latest canary version 0.0.39

**Comment by FreeSlave** [comment]
Yep, Discord gave me the right link when it asked to update to 0.0.39.
